composer.json

```diff
@@ -14,22 +14,24 @@
     ],
     "autoload": {
         "psr-4": {
-            "Dhl\\Paket\\": ""
+            "Dhl\\Paket\\Api\\": "Api/",
+            "Dhl\\Paket\\Model\\": "Model/",
+            "Dhl\\Paket\\Observer\\": "Observer/"
         },
         "files": [
             "registration.php"
         ]
     },
     "require": {
-        "php": "^7.2.0 || ^8.1.0",
-        "dhl/module-carrier-paket": "^2.0.0",
+        "php": "^7.2.0 || ^8.1.0 || ^8.2.0",
```

Comment:
Magento 2.4.7 also supports PHP 8.3 according to [Docs](https://experienceleague.adobe.com/en/docs/commerce-operations/installation-guide/system-requirements)

Reply:
Couldn't test that. 
So not making any such claims in composer.

Reply:
We are also primarily interested in a PHP8.2 update. Any plan here ?

Reply:
If someone can give it a test with 8.3 and confirm it works, that can be added too.
We're only set up to test planned updates to our production system and 8.3 will come later due to other plugins.

Reply:
Just fyi: the change of the PHP requirement is not necessary, as both, PHP 8.2 and 8.3 are already valid for the existing requirement (see [here](https://jubianchi.github.io/semver-check/#/^8.1.0/8.3.x))